Tools/CISupport/ews-build/events.py

```diff
@@ -356,7 +356,25 @@ class GitHubEventHandlerNoEdits(GitHubEventHandler):
     _last_commit_classes_refresh = 0
     COMMIT_CLASSES_REFRESH = 60 * 60 * 24  # Refresh our commit-classes once a day
     COMMIT_CLASSES_URL = f'https://raw.githubusercontent.com/{PUBLIC_REPOS[0]}/main/metadata/commit_classes.json'
+    DUPE_DETECTOR = {}
 
+    # This approach only works as long as we have a single build-master responding to hooks
+    @classmethod
+    def is_duped(cls, value, bucket=60):
```

Comment:
Might be a good idea to add some comments in this function describing what/how we are doing here.